### ISSUE TITLE
Bile duct polyp

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7638]
**Preferred term label:**
Bile duct polyp

**Synonyms**

**Definition (free text, please give PubMed ID)**
An elevated growth from the the mucosa of the bile duct.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
175200 (OMIM)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
